backend/src/main/kotlin/com/tngtech/apicenter/backend/config/RestResponseExceptionHandler.kt

```diff
@@ -63,6 +63,10 @@ class RestResponseExceptionHandler {
 
     @ExceptionHandler(RemoteFileConnectionRefusedException::class)
     fun handleConnectionRefused(exception: RemoteFileConnectionRefusedException) =
-            makeResponseEntity("Specification couldn't be retrieved from ${exception.location}.", HttpStatus.NOT_FOUND)
```

Comment:
Why not keep the error message?

Reply:
Is what's written in the PR description ok justification?

Reply:
Oh yes, but I should read it :p